README.md

```diff
@@ -21,6 +21,14 @@ git clone git@github.com:lagom/online-auction-java.git
 cd online-auction-java
 sbt runAll
 ```
+## Importing into IDEs
+
+### Intellij IDEA
+You will need to install following plugins into IDEA by going to `File -> Settings -> Plugins`
+* **[SBT](http://github.com/orfjackal/idea-sbt-plugin)** : SBT build tool plugin.
```

Comment:
Hey @ignasi35 So will it be good to 
- Get abstract steps from there with a link to that documentation?
or 
- We should only link that documentation?